src/frontend/components/component-info/component-info.ts

```diff
@@ -22,44 +23,21 @@ import {
   template: require('./component-info.html'),
 })
 export class ComponentInfo {
-  @Input() node: Node;
-  @Input() tree: MutableTree;
-  @Input() state;
-  @Input() metadata: Metadata;
-  @Input() loadingState: ComponentLoadState;
+  @Input() private node: Node;
+  @Input() private tree: MutableTree;
+  @Input() private state;
+  @Input() private metadata: Metadata;
+  @Input() private loadingState: ComponentLoadState;
 
   @Output() private selectNode = new EventEmitter<Node>();
 
   private ComponentLoadState = ComponentLoadState;
 
-  path: Path;
-
-  inputs: InputOutput;
-  outputs: InputOutput;
-
-  ngOnInit() {
-    this.ngOnChanges();
-  }
+  private path: Path;
 
-  ngOnChanges() {
+  ngOnChanges(changes: SimpleChanges) {
```

Comment:
@clbond I don't see changes being used. Is it added just to conform with the signature of the method?